apps/tmbitwarden.sh

```diff
@@ -79,38 +79,92 @@ Basically:
 Please have a look at how the questions are answered here if you are uncertain:
 https://i.imgur.com/YPynDAf.png"
 
+# Create bitwarden user
+if ! id "$BITWARDEN_USER" >/dev/null 2>&1
+then
+    print_text_in_color "$ICyan" "Specifying a certain user for Bitwarden: $BITWARDEN_USER..."
+    useradd -s /bin/bash -d "$BITWARDEN_HOME" -m -G docker "$BITWARDEN_USER"
+else
+    userdel "$BITWARDEN_USER"
+    rm -rf "${BITWARDEN_HOME:?}/"
+    print_text_in_color "$ICyan" "Specifying a certain user for Bitwarden: $BITWARDEN_USER..."
+    useradd -s /bin/bash -d "$BITWARDEN_HOME/" -m -G docker "$BITWARDEN_USER"
+fi
+
+if [ ! -d "$BITWARDEN_HOME" ]
+then
+    mkdir -p "$BITWARDEN_HOME"
+    chown -R "$BITWARDEN_USER":"$BITWARDEN_USER" "$BITWARDEN_HOME"
+fi
+
+# Create the service
+msg_box "We will now create a systemd service for Bitwarden.
+
+START Bitwarden, simply execute: systemctl start bitwarden
+STOP Bitwarden, simply execute: systemctl stop bitwarden
+RESTART Bitwarden, simply execute: systemctl restart bitwarden"
```

Comment:
Hm, I think I would put this to the end of the script...
There are already too many msg_boxes at the beginning of the script

Reply:
I've moved it to the buttom.